app/components/ui/HamburgerMenu.tsx

```diff
@@ -56,10 +59,26 @@ export const HamburgerMenu = ({
     width="275px"
   >
     {links.map(({ to, text, exact = false }) => (
-      <MenuItem key={to} to={to} onClick={toggleHamburgerMenu} exact={exact}>
+      <MenuItem
+        key={to}
+        to={to}
+        onClick={toggleHamburgerMenu}
+        exact={exact}
+        linkStyles={styles.menuItem}
+      >
         {text}
       </MenuItem>
     ))}
+    {showReportCrisis && (
+      <MenuItem
+        to="https://sf.gov/information/reporting-concerns-about-street-crises-and-conditions"
+        onClick={toggleHamburgerMenu}
+        exact={false}
+        linkStyles={`${styles.menuItem} ${styles.emphasized}`}
+      >
+        Report Street Crisis
+      </MenuItem>
```

Comment:
I'm wondering if it'd be better to (conditionally) add this to the `links` array and just add extra properties for the things that are special about this item. This would keep this component data-driven and more consistent with the other link items we have.

Reply:
On a separate note, it looks like Dongyeon responded to your question on Figma, and he does think that these links should open in another window/tab, so we probably need to add an extra property to control that.

Reply:
Sounds good. Should I update the other external links to open in a new tab as well?

Reply:
I think the Twitter and Facebook links would make sense to open in a new tab, but ones under the help.sfserviceguide.org domain maybe should stay in the same tab, since even though it takes you to an external site, it's still conceptually part of the same site? But I don't think I have a super strong opinion about this and could go either way.